src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@parjong, since you've said you are trying to stay with keeping the existing unwind info embedded in x86 GC info, I am not sure I understand the addition of the code to Compiler::unwindXXXX methods. Shouldn't they just be empty as before?